docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+
+volumes:
+    data:
+        driver: local
+    cache:
+        driver: local
+    logs:
+        driver: local
+
+services:
+    sylius:
+        build:
+            context: .
+            dockerfile: ./etc/docker/php/Dockerfile
+        entrypoint: /var/www/etc/docker/php/entrypoint.sh
+        volumes:
+            - .:/var/www
+            - data:/var/www/sylius
```

Comment:
I must have misread your previous file, I thought this was some kind of "uploads" folder. My bad :)